test_fn_grad_linalg_det_singular_cpu_float64 fails not only on macos

Author: Alexander Grund (TU Dresden)

diff --git a/torch/testing/_internal/opinfo/definitions/linalg.py b/torch/testing/_internal/opinfo/definitions/linalg.py
index 616c8cf42f4..3a07d19df46 100644
--- a/torch/testing/_internal/opinfo/definitions/linalg.py
+++ b/torch/testing/_internal/opinfo/definitions/linalg.py
@@ -1135,7 +1135,6 @@ op_db: List[OpInfo] = [
                 "test_fn_grad",
                 device_type="cpu",
                 dtypes=(torch.float64,),
-                active_if=IS_MACOS,
             ),
             DecorateInfo(
                 unittest.skip("Gradients are incorrect on macos"),
